src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,103 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin("Login");
+        user.setPassword("password");
+        user.setAge(30);
+    }
+
+    @Test
+    void register_userCorrectValue_Ok() {
+        registrationService.register(user);
+        User newUser = storageDao.get(user.getLogin());
+        assertNotNull(storageDao.get(user.getLogin()),
+                "Dao should return not null value");
+        assertEquals(newUser.getLogin(), user.getLogin(),
+                "Expected user login is " + user.getLogin()
+                        + ", but was " + newUser.getLogin() + ".");
+        assertEquals(newUser.getPassword(), user.getPassword(),
+                "Expected user password is " + user.getPassword()
+                        + ", but was " + newUser.getPassword() + ".");
+        assertEquals(newUser.getAge(), user.getAge(),
+                "Expected user age is " + user.getAge()
+                        + ", but was " + newUser.getAge() + ".");
```

Comment:
For the future, no need to write a message, it will be generated automatically
```suggestion
        assertNotNull(storageDao.get(user.getLogin()));
        assertEquals(newUser.getLogin(), user.getLogin());
        assertEquals(newUser.getPassword(), user.getPassword());
        assertEquals(newUser.getAge(), user.getAge());
```